tox.ini

```diff
@@ -53,33 +53,30 @@ deps =
     devdeps: git+https://github.com/astropy/astropy
     devdeps: git+https://github.com/matplotlib/matplotlib
 
+    # The 32bit and py39 build needs numpy installed before trying to install sunpy due
+    # to https://github.com/scikit-image/scikit-image/issues/4261
+    32bit,py39: numpy
+
     # Oldest deps we pin against.
-    oldestdeps: astropy<4.1
+    oldestdeps: astropy<4.2
     oldestdeps: numpy<1.17.0
     oldestdeps: matplotlib<3.2
-    oldestdeps: scipy<1.3
-    oldestdeps: parfive<1.2
+    oldestdeps: scipy<1.4
     oldestdeps: pandas<0.25
-    oldestdeps: parfive<1.2
-    oldestdeps: drms<0.7
```

Comment:
Why remove drms?

Reply:
For DRMS and parfive, since we control those directly, I would rather try to pin them to newest ones as possible. But I can add them back.

Reply:
Well this PR pins parfive to the newest release anyway, but probably worth keeping around here for the future.

Reply:
I think we should add them back, as there will always be a minimum version that we support, even if it's the latest version at the time of a sunpy release.

Reply:
Added it back.